llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -1966,6 +1966,16 @@ bool RISCVTTIImpl::isLSRCostLess(const TargetTransformInfo::LSRCost &C1,
                   C2.ScaleCost, C2.ImmCost, C2.SetupCost);
 }
 
+bool RISCVTTIImpl::isLegalMaskedExpandLoad(Type *DataTy, Align Alignment) {
+  auto *VTy = dyn_cast<VectorType>(DataTy);
+  if (!VTy || VTy->isScalableTy())
+    return false;
```

Comment:
```suggestion
  if (!isa<ScalableVectorType>(DataTy))
    return false;
```

Reply:
I think we don't want scalable types here.

Reply:
Oh woops, that should be inverted